tests/operator/operator.go

```diff
@@ -3352,3 +3343,13 @@ func deleteAllVMIs(vmis []*v1.VirtualMachineInstance) {
 		Expect(err).ToNot(HaveOccurred(), "Delete VMI successfully")
 	}
 }
+
+func getVirtLauncherSha(deploymentConfigStr string) (string, error) {
```

Comment:
```
Change the getVirtLauncherSha lambda to be a regular func
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>
```

in the commit message, please mention the change of function signature, and why it is important.